hlf/scala/admission.md

```diff
@@ -0,0 +1,49 @@
+# Hyperledger Scala API Admission
+
+In the following, the hlf-api for the "admission" handling will be explained.
+For insight in the General hlf-api, please refer to it's .Readme - File.
+> [!NOTE]
+Any Method can throw these [Exceptions](errors.md#Exceptions).
+
+For "admission" handling we offer the following interface. 
+Additionally, as described in [General Communication](general-communication.md), different proposal calls can be made.
+
+
+## AddAdmission(enrollmentId: String, courseId: String, moduleId: String, timestamp: String)

```

Comment:
Change this to single admissionObject as well?

Reply:
resolved in https://github.com/upb-uc4/api/pull/84/commits/04d7b135164637c34443487fa345f0c49e8f5d30